pkgs/tools/audio/liquidsoap/full.nix

```diff
@@ -7,18 +7,23 @@
 
 let
   pname = "liquidsoap";
-  version = "2.2.5";
+  version = "2.3.x-unstable-2024-09-02";
```

Comment:
What is .x here doing?

Reply:
In that case, I think 2.3-unstable... would be more appropriate.

Reply:
It’s a bit subtle – usually `VERSION-unstable-YYYY-MM-DD` means an unstable version *after* `VERSION`. In this case, the version was after the birth of the `rolling-release-v2.3.x` branch and the marking of the 2.3.x release “series” as being available in alpha, but before the release of 2.3.0. So `2.3-unstable-…` feels a little misleading to me, as it would be prior to the release of 2.3.0, and we could conceivably use e.g. `2.3.0-unstable-…` to designate a commit from the rolling release branch *after* the release of 2.3.0. (Anyway, I dropped this from the PR.)